pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_scaler.go

```diff
@@ -123,7 +123,7 @@ func (ks *kpaScaler) Scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, de
 	// Identify the current scale.
 	scl, err := ks.scaleClientSet.Scales(pa.Namespace).Get(resource, resourceName)
 	if err != nil {
-		logger.Errorf("Resource %q not found.", resourceName, zap.Error(err))
+		logger.Errorf("Resource %q not found: %v", resourceName, zap.Error(err))
```

Comment:
Same comment on `Errorw`, we might need to use `fmt.Sprintf` though. @vagababov?

Reply:
Done.